src/3_branch.js

```diff
@@ -149,6 +150,19 @@ Branch.prototype._referringLink = function() {
 	else { return null; }
 };
 
+/***
+ * @function Branch._publishEvent
+ * @param {string} event
+ * @param {string} subject
+ */
+Branch.prototype._publishEvent = function(event, subject) {
```

Comment:
yea - we can just do one at a time for now:
```
branch.addListener("willCloseBanner", fn);
branch.addListener("didCloseBanner", fn);
```
also - the typical syntax is to put the event first, the function second.